var/spack/repos/builtin/packages/charliecloud/package.py

```diff
@@ -17,15 +17,22 @@ class Charliecloud(AutotoolsPackage):
     version('master', branch='master')
     version('0.15',   sha256='2163420d43c934151c4f44a188313bdb7f79e576d5a86ba64b9ea45f784b9921')
     version('0.14',   sha256='4ae23c2d6442949e16902f9d5604dbd1d6059aeb5dd461b11fc5c74d49dcb194')
+    version('0.13',   sha256='5740bff6e410ca99484c1bdf3dbe834c0f753c846d55c19d6162967a3e2718e0')
 
     depends_on('m4',       type='build')
     depends_on('autoconf', type='build')
     depends_on('automake', type='build')
     depends_on('libtool',  type='build')
 
-    depends_on('python@3.5:',    type='run')
-    depends_on('py-lark-parser', type='run')
-    depends_on('py-requests',    type='run')
+    # Use legacy build system up to version 0.13
+    depends_on('skopeo',         type='run', when='@:0.13')
+    depends_on('umoci',          type='run', when='@:0.13')
+    depends_on('python+libxml2', type='run', when='@:0.13')
```

Comment:
These dependencies should be from 0.10 to 0.13. Prior to 0.10 ch-grow wasn't offered and we used a manual make file build. However, I'm not sure how much that matters here; it will only be an issue if more older versions are added.

Reply:
I went ahead and added this just to be on the safe side. It won't hurt anything to add it in there, and will make it easier if someone needs older versions in the future.